src/data/regex-quiz.ts

```diff
@@ -1,4 +1,31 @@
 const regexQuiz = [
+  {
+    Question: "What does the regex character class [A-Za-z] match?",
+    Answer: "Any letter(both uppercase and lowercase)",
+    Distractor1: "Any uppercase letter",
+    Distractor2: "Any lowercase letter",
+    Distractor3: "Any single character",
+    Explanation: "This character class matches any single letter, whether it's uppercase or lowercase.",
+    Link: "https://www.freecodecamp.org/news/practical-regex-guide-with-real-life-examples/"
+  },
+  {
+    Question: "Which regex metacharacter represents the start of a line?",
+    Answer: "^",
+    Distractor1: "$",
+    Distractor2: ".",
+    Distractor3: "\\",
+    Explanation: "The ^ metacharacter represents the start of a line in a regular expression.",
+    Link: "https://www.freecodecamp.org/news/practical-regex-guide-with-real-life-examples/"
+  },
+  {
+    Question: "Which regex flag is used to make the pattern-matching case-insensitive?",
+    Answer: "i",
+    Distractor1: "m",
+    Distractor2: "g",
+    Distractor3: "s",
+    Explanation: "The i flag makes the regex pattern matching case-insensitive so it will match both uppercase and lowercase characters.",
+    Link: "https://www.freecodecamp.org/news/practical-regex-guide-with-real-life-examples/"
+  },
```

Comment:
both of these questions already exist in the quiz file.
Please remove both of them

Reply:
Yeah sure, Can you please hacktoberfest-accepted label to my PR? Because the first two contributions were accepted by hacktoberfest  but not the last two.

Reply:
@sahilbheke 

>Can you please hacktoberfest-accepted label to my PR?

So the hacktoberfest accepted label is applied after a maintainer approves the PR.
So this PR will get that label after I approve and will count towards your total.

>but not the last two

The last two PRs here were closed because they were duplicate questions and will not be accepted into the project.
https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/759
https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/758

Those PR's will not get the hacktoberfest accepted label because they were not approved and will not go into the main branch.

Here is the official Hacktoberfest rules found on the [participation page](https://hacktoberfest.com/participation/)

Project maintainers must accept your pull/merge requests for them to count toward your total

Hope that helps 👍

Reply:
Thank you for helping.